utils/single_test.py

```diff
@@ -3,18 +3,18 @@
 Convert a single line from test output to run single test command.
 
 E.g:
-    "2: test_download_package_via_metalink (tests.test_yum_package_downloading.TestCaseYumPackageDownloading) ... ok"
+    "test_download_package_via_metalink (tests.test_yum_package_downloading.TestCaseYumPackageDownloading) ... ok"
 
 To:
-    tests/python/tests/test_yum_repo_downloading.py:TestCaseYumRepoDownloading.test_download_and_update_repo_01
+    "python3 -m unittest discover -s tests/python/ -p tests/test_yum_repo_downloading.py -k test_download_and_update_repo_01"
```

Comment:
... aaand a copy-paste. Nice :smirk: (context: https://github.com/rpm-software-management/createrepo_c/pull/246#discussion_r575123728)

Reply:
I also removed it here.